app/js/components/orderEntry/OrdersTable.jsx

```diff
@@ -99,17 +212,52 @@ OrdersTable.propTypes = {
   }).isRequired,
   status: PropTypes.objectOf(PropTypes.bool).isRequired,
   dateFormat: PropTypes.string,
+  allConfigurations: PropTypes.shape({
+    drugDispensingUnits: PropTypes.arrayOf(PropTypes.any),
+    drugDosingUnits: PropTypes.arrayOf(PropTypes.any),
+    drugRoutes: PropTypes.arrayOf(PropTypes.any),
+    durationUnits: PropTypes.arrayOf(PropTypes.any),
+    orderFrequencies: PropTypes.arrayOf(PropTypes.any),
```

Comment:
@Efosaok when you say it's generally not advised, can you show me where ? I know it doesn't give the actual representation of a prop and it also helps for documentation purposes also but it's been used extensively in this particular code base and as such I deduced that it won't be an issue using it here because we have no rule against using it sort of like the wildcard import, but if you were to stumble upon it, I'd be happy to get rid of it.

Reply:
The rule that forbids `PropType.any` prop-declaration might not be configured for this project but that doesn't mean that if we do not have a rule that enforces we use a semicolon, we would not use it.
I have given a reason above why the `PropType.any` declaration is not sufficient, but if you feel the reason is not valid enough to warrant a change, I have no problem with that. Just as you rightly said up there; `we have no rule against using it`     😄